### PR TITLE
ALPC logo

### PR DESCRIPTION
**Pull request by oncletom**
And MIT license in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,13 +98,34 @@ Thanks to [@pascalc](https://twitter.com/pascalchevrel), [@guillaumemasson](http
...
+![ALPC logo](logo-alpc.jpg).
 
-This grant is being shared among David Bruant (5000€) and Matthieu Bué (5000€) to get the project to a robust, easy-to-use, well-designed and featurefull 1.0 version.
+This grant is being shared among David Bruant (5,000€) and Matthieu Bué (5,000€) to get the project to a robust, easy-to-use, well-designed and featurefull 1.0 version.
 
 
 # Licence
 
-MIT
+> Copyright © 2016 David Bruant
```
Comment (DavidBruant) [inline README.md]: There is already a licence file in the repo, no need to repeat the text here. At best, add a link to the repo's file.

Comment (oncletom) [inline README.md]: Okay will remove that. It is actually unrelated to the PR so will revert this change.

**Comment by DavidBruant** [comment]
Thanks!
